### ISSUE TITLE
Cannot Complete Install

### ISSUE THREAD

**Issue by godtoall82** [gamer-os/gamer-os#200]
So the OS downloads and installs just fine but when I get to the steam update screen, I get “Fatal Error, steam must be connected to the internet...”. I’m connected via Ethernet so not sure what the issue is.

**Comment by Pinkys-Paw** [comment]
Ditto in every respect, I wonder if it's Steam being ... difficult.

**Comment by Pinkys-Paw** [comment]
> So the OS downloads and installs just fine but when I get to the steam update screen, I get “Fatal Error, steam must be connected to the internet...”. I’m connected via Ethernet so not sure what the issue is.

Well, I checked back in today, as I was about to crack and <shudder> buy a copy of Windows ... and it completed the install just fine. So no idea what that was about, I've been checking in weekly up until about two weeks ago and still no dice. I also read back through some old tickets and there was mention of poss network card issues ... so I did wonder if it was the age of my motherboard (and hence network connection) ... but I guess not! 

Plugged in my PS$ controller and we're off. I'm most pleased :)
